xonsh/xontribs.json

```diff
@@ -170,71 +170,71 @@
    "url": "http://exofrills.org",
    "install": {
     "conda": "conda install -c conda-forge xo",
-    "pip": "pip install exofrills"}
+    "pip": "xip install exofrills"}
```

Comment:
I don't know that all of the instances in this file should be changed, should they?